README.md

```diff
@@ -57,3 +57,36 @@ pip install -e .
 
 The `mcmd` command will now be available in this virtual environment! If you want to
 leave the environment, use `deactivate`.
+
+### Running tests
+The project contains unit- and integration tests. 
+
+#### Running tests from the command line
+To run all the unit tests from the command line, go:
+
+```
+python setup.py test --addopts "-m unit --ignore=tests/integration/" 
+```
+
+The integration tests require a running MOLGENIS application to test against. They can be run like this:
+ 
+```
+python setup.py test --addopts "-m integration --ignore=tests/unit/"
+```
+ 
+By default the localhost is chosen (username: admin, password: admin), but it's possible to configure a different server:
+
+```
+python setup.py test --addopts "-m integration --ignore=tests/unit/ --url=<your_url> --username=<admins username> --password=<admins password>" 
+```
+
+#### Running tests in PyCharm
+To run the tests in PyCharm, first set the default test runner to 'pytest'. 
+
+![Configure default test runner](docs/default_test_runner.png)
+
+Then create pytest run configurations using the same arguments as above:
+
+![Setting up run configuration](docs/run_configuration.png)
```

Comment:
```C:\Users\Dennis\Dev\molgenis-tools-commander\env\Scripts\python.exe "C:\Program Files\JetBrains\PyCharm 2018.3.4\helpers\pycharm\_jb_unittest_runner.py" --path C:/Users/Dennis/Dev/molgenis-tools-commander -- -m unit --ignore=tests/integration/
usage: python -m unittest discover [-h] [-v] [-q] [--locals] [-f] [-c]
                                   [-k TESTNAMEPATTERNS] [-s START]
                                   [-p PATTERN] [-t TOP]
python -m unittest discover: error: unrecognized arguments: -m --ignore=tests/integration/
Launching unittests with arguments python -m unittest discover -s C:/Users/Dennis/Dev/molgenis-tools-commander -t C:\Users\Dennis\Dev\molgenis-tools-commander -m unit --ignore=tests/integration/ in C:\Users\Dennis\Dev\molgenis-tools-commander


Process finished with exit code 2

Reply:
I think you missed an argument value, it should say:

`-m unit --ignore=tests/integration/`

Reply:
The argument was ok. Turns out I had to create a 'Python test' of type 'pytest' instead of unit test.